src/components/application_manager/test/commands/mobile/set_audio_streaming_indicator_test.cc

```diff
@@ -0,0 +1,526 @@
+      mobile_apis::Result::APPLICATION_NOT_REGISTERED;
+  const bool is_success = kIsNotSuccess;
+
+  ApplicationSharedPtr mock_app_empty;
+  ON_CALL(app_mngr_, application(kConnectionKey))
+      .WillByDefault(Return(mock_app_empty));
+
+  TestCommandRunMethod(
+      msg_mobile_response, mobile_response, is_success, command);
+}
+
+TEST_F(SetAudioStreamingIndicatorRequestTest, Run_NaviApplication_REJECTED) {
+  MessageSharedPtr msg_mobile = CreateFullParamsSO();
+  utils::SharedPtr<SetAudioStreamingIndicatorRequest> command =
+      CreateCommand<SetAudioStreamingIndicatorRequest>(msg_mobile);
+
+  MessageSharedPtr msg_mobile_response = CreateFullParamsSO();
+  const mobile_apis::Result::eType mobile_response =
+      mobile_apis::Result::REJECTED;
+  const bool is_success = kIsNotSuccess;
+
+  NaviAppSetup();
+
+  TestCommandRunMethod(
+      msg_mobile_response, mobile_response, is_success, command);
+}
+
+TEST_F(SetAudioStreamingIndicatorRequestTest,
+       Run_NotMediaApplication_REJECTED) {
+  MessageSharedPtr msg_mobile = CreateFullParamsSO();
+  utils::SharedPtr<SetAudioStreamingIndicatorRequest> command =
+      CreateCommand<SetAudioStreamingIndicatorRequest>(msg_mobile);
+
+  MessageSharedPtr msg_mobile_response = CreateFullParamsSO();
+  const mobile_apis::Result::eType mobile_response =
+      mobile_apis::Result::REJECTED;
+  const bool is_success = kIsNotSuccess;
+
+  IncorrectAppSetup();
+
+  TestCommandRunMethod(
+      msg_mobile_response, mobile_response, is_success, command);
+}
+
+TEST_F(SetAudioStreamingIndicatorRequestTest,
+       Run_VoiceCommunicationApplication_REJECTED) {
+  MessageSharedPtr msg_mobile = CreateFullParamsSO();
+  utils::SharedPtr<SetAudioStreamingIndicatorRequest> command =
+      CreateCommand<SetAudioStreamingIndicatorRequest>(msg_mobile);
+
+  MessageSharedPtr msg_mobile_response = CreateFullParamsSO();
+  const mobile_apis::Result::eType mobile_response =
+      mobile_apis::Result::REJECTED;
+  const bool is_success = kIsNotSuccess;
+
+  VoiceCommunicationAppSetup();
+
+  TestCommandRunMethod(
+      msg_mobile_response, mobile_response, is_success, command);
+}
+
+TEST_F(SetAudioStreamingIndicatorRequestTest,
+       Run_AddIndicatorWaitingForResponseFailed_IGNORED) {
+  MessageSharedPtr msg_mobile = CreateFullParamsSO();
+  utils::SharedPtr<SetAudioStreamingIndicatorRequest> command =
+      CreateCommand<SetAudioStreamingIndicatorRequest>(msg_mobile);
+
+  MessageSharedPtr msg_mobile_response = CreateFullParamsSO();
+  const mobile_apis::Result::eType mobile_response =
+      mobile_apis::Result::IGNORED;
+  const bool is_success = kIsNotSuccess;
+
+  MediaAppSetup();
+
+  ON_CALL(*mock_app_, AddIndicatorWaitForResponse(_))
+      .WillByDefault(Return(false));
+
+  TestCommandRunMethod(
+      msg_mobile_response, mobile_response, is_success, command);
+}
+
+TEST_F(SetAudioStreamingIndicatorRequestTest,
+       Run_SetAudioStreamingIndicatorRequestOK_SendToHmi) {
```

Comment:
@tkireva, please, rename test cases correctly. `SendToHMI` in not clear. What exactly should be send to `HMI`?